WTWSMS/common/government_flavor/WtWSMS_government_flavor.txt

```diff
@@ -34,4 +37,26 @@ government_flavor = {
 	trigger = {
 		government = legion_government
 	}
-}
\ No newline at end of file
+}
+government_flavor = {
+	name = gov_bureaucracy
+	graphics = GFX_government_flavor_bureaucracy_placeholder
```

Comment:
I think there is a big difference between Roman bureaucracies chopping off heads and a placeholder surname.